Jellyfin.Plugin.KodiSyncQueue/Data/DbRepo.cs

```diff
@@ -141,11 +142,10 @@ public void SetUserInfoSync(List<MediaBrowser.Model.Dto.UserItemDataDto> dtos, L
             var newRecs = new List<UserInfoRec>();
             var upRecs = new List<UserInfoRec>();
             var userInfoCollection = _liteDb.GetCollection<UserInfoRec>(UserInfoCollection);
-            var jsonOptions = JsonDefaults.GetOptions();
 
             dtos.ForEach(dto =>
             {
-                var sJson = System.Text.Json.JsonSerializer.Serialize(dto, jsonOptions);
+                var sJson = System.Text.Json.JsonSerializer.Serialize(dto, _jsonSerializerOptions);
                 _logger.LogDebug("Updating ItemId '{0}' for UserId: '{1}'", dto.ItemId, userId);
 
                 LibItem itemref = itemRefs.FirstOrDefault(x => x.Id.ToString("N", CultureInfo.InvariantCulture) == dto.ItemId);
```

Comment:
Serialize inside the if statement, use `{@Name}` to serialize to json in the log
Parse the Guid outside the LINQ statement and compare with that

Reply:
Not quite sure of what you mean by `{@Name}` here.

This whole plugin is due for a full do-over with a fine-toothed comb for optimizations. I do think that belongs in another PR or three however :wink: 

Getting analyzers going is the first step of me hopefully not screwing up (as bad) when I start poking around some more :see_no_evil:

Reply:
Nvm, the json string isn't even logged